testing/test_runner.py

```diff
@@ -482,15 +482,28 @@ def test_callinfo():
     assert ci.when == "123"
     assert ci.result == 0
     assert "result" in repr(ci)
-    assert repr(ci) == "<CallInfo when='123' result: 0>"
+    assert repr(ci) == "<CallInfo when='123' _result=0>"
```

Comment:
"result: 0" looks nicer in general, but the new form provides the attribute verbatim already.